.github/workflows/self-hosted.yml

```diff
@@ -65,6 +65,20 @@ jobs:
         run: make bootstrap-cluster
         working-directory: cnf-certification-test-partner
 
+      - name: Prune the local docker system
+        uses: nick-fields/retry@v2
+        with:
+          timeout_minutes: 10
+          max_attempts: 3
+          command: docker system prune -f
+
+      - name: Preemptively delete the Kind cluster
```

Comment:
I would delete the kind cluster first. Then, docker system prune to remove everything that is left.

Reply:
Good call, I'll do that.